URA-695/compare_vcfs.sh

```
@@ -0,0 +1,14 @@
+
+for vcf_file in sentieon-tnbam_v3.2.0/*markdup_recalibrated_tnhaplotyper2.vcf.gz
+    do sample=$(basename "$vcf_file" .vcf.gz)
+    echo $sample
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)